jquery.balancetext.js

```diff
@@ -282,7 +282,11 @@
     $.fn.balanceText = function (skipResize) {
         var selector = this.selector;
 
-        if (!skipResize && balancedElements.indexOf(selector) === -1) {
+        if (!selector && this[0].className) {
+            selector = "." + this[0].className;
```

Comment:
This fixes the constrained case in test1.htm file, but it creates more problems as it fixes, so I decided to remove this part of the code.